lib/kitchen/document.rb

```diff
@@ -131,11 +137,7 @@ def create_element(name, *args, &block)
     # @return [Element]
     #
     def create_element_from_string(string)
-      Kitchen::Element.new(
-        node: @nokogiri_document.create_element(string),
-        document: self,
-        short_type: "created_element_#{SecureRandom.hex(4)}"
-      ).element_children.first
+      new_element(html: string)
```

Comment:
Is this calling the spec helper?  Don't do that.  Specs can depend on real code but real code should not depend on specs.